### PR TITLE
Copy Credential Management API compat data from existing article

### PR DESCRIPTION
**Pull request by Asgavar**

### CODE REVIEW

File: api/CredentialManagementAPI.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            },
+            "chrome": {
+              "version_added": "65",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "enable-webauthentication",
+                  "value_to_set": "true"
+                }
+              ]
```
Comment (jpmedley) [inline api/CredentialManagementAPI.json]: Chrome enabled this by default in 67. https://www.chromestatus.com/features/5669923372138496

File: api/CredentialManagementAPI.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/CredentialManagementAPI.json]: Chrome 70. https://www.chromestatus.com/features/5669923372138496

File: api/CredentialManagementAPI.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+          }
+        }
+      },
+      "web_authentication": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Web_Authentication_API",
+          "description": "Web authentication",
+          "support": {
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/CredentialManagementAPI.json]: Should be `false`.

**Comment by Elchi3** [comment]
Thanks for you pull request. The Web Credential compat data has actually been converted to data in this repository already. See the interface pages, like https://developer.mozilla.org/en-US/docs/Web/API/Credential. It's just that this overview page https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API still contains the old table, but we don't want to migrate it again. I've filed https://github.com/mdn/kumascript/issues/958 to remove the call to help. The old table will be removed soon hopefully. Sorry for the inconveniences.

**Pull request closed without merging**